File: DCP-03-26/Find-Unique-Binary-String.cpp
1class Solution {
2public:
3    string findDifferentBinaryString(vector<string>& nums) {
4        int n = nums.size();
5        string ans = "";
6        for(int i=0;i<n;i++){
7            char ch = nums[i][i];
8            ans += (ch=='0')? "1" : "0"; 
9        }
10        return ans;
11    }
12};

File: DCP-03-26/Find-Unique-Binary-String.java
1class Solution {
2    public String findDifferentBinaryString(String[] nums) {
3        StringBuilder result = new StringBuilder();
4        for (int i = 0; i < nums.length; i++) {
5            if (nums[i].charAt(i) == '0') {
6                result.append('1');
7            } else {
8                result.append('0');
9            }
10        }
11        return result.toString();
12    }
13}